frontend/src/vault/chain/VaultChainPage.tsx

```diff
@@ -55,13 +62,32 @@ export const VaultChainPage = () => {
 
   const nativeCoin = useAssertCurrentVaultNativeCoin(chainId);
 
+  const { mutate: refresh, isPending } = useMutation({
+    mutationFn: () => {
+      return invalidateQueries(
+        getBalanceQueryKey({
```

Comment:
There are might be other coins beside the native one on that page, right? Then you should invalidate other coins as well.